Aptoide/aptoide-client-v8
# [Fix/ASV-1264] Wallet claimed not installed

**What does this PR do?**

   Locks other games when wallet is not installed but claimed.
   Shows install instead of install to get APPC when wallet is claimed but not installed.
   Shows claimed after installing an already claimed wallet.
   Fixed some layout problems like promotions view Toolbar title broken in other languages, Claim Dialog Title margins & button width's.


**Database changed?**

   No

**Where should the reviewer start?**

- [ ] PromotionsFragment.java

**How should this be manually tested?**

  Go to promotions with claimed wallet but not installed - check if other games are locked.
  See if wallet says install
  Install wallet and see if view updates to claimed.
  Layout issues must be tested by localisation in the multi-language apk.  

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1264](https://aptoide.atlassian.net/browse/ASV-1264)


**Code Review Checklist**
